docs/design/crds.md

```diff
@@ -149,4 +149,117 @@ spec:
       bufferSize: 12 # Size of a log sending buffer in kilobytes
       syslogConfig: 
         name: syslog-config-map # Name of a config map containing syslog connection configuration
+```
+
+## EdgeConfig
+`EdgeConfig` is a namespaced custom resource that represents custom configuration that should be deployed to edge devices matching criteria specified in the CR.
+
+* apiVersion: `management.project-flotta.io/v1alpha1`
+* kind: `EdgeConfig`
+
+### Specification
+```yaml
+spec:
+  deviceIDs: # The deviceID list on which the playbook should be executed. Necessary to execute playbook on a devices that don't belong to any group
```

Comment:
this seems outdated. no mention of `deviceIDs` in the PR.

Reply:
Done